### DISCUSSION TITLE
Can I set a maximum height?

### DISCUSSION THREAD

**Discussion by AndreRX** [GriefPrevention/GriefPrevention#1104] [Q&A] [answered]
when a player claim a land, the height is unlimited, i want a maximum of 30 block.

How can i do that?

**Comment by bigpresh** [comment]
GP claims have no max height.

It may be possible for an addon to change that, but I'm not sure, would have to look at the code closely.

I don't imagine that such an option would be likely in GP core, though - it'd add confusion and not be of use to many people.

**Comment by Jikoo** [comment, marked as answer]
Duplicate of #269

**Comment by Jikoo** [comment]
I know this specifically requests main claim and not subclaims, but GP treats them roughly the same way, so the same large changes to move from 2D to 3D space would be required.
